codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -79,6 +80,27 @@ public void streamInactive(Http2Stream stream) {
             }
 
             @Override
+            public void streamHalfClosed(Http2Stream stream) {
+                if (State.HALF_CLOSED_LOCAL.equals(stream.state())) {
+                    /**
+                     * Any pending frames can never be written, clear and
```

Comment:
@nmittler  oh yeah that's still a relict. I just copied it from `streamClosed`.

Fun fact, git blame tells me you are the original author :-)).